src/themes/shared/tokens/sizes.ts

```diff
@@ -28,3 +29,13 @@ export const sizes = defineTokens.sizes({
   'auth0-height': { value: '33.75rem' },
   'screen-height': { value: '100vh' },
 });
+
+export type SizeToken =
+  | 'sm'
+  | 'md'
+  | 'lg'
+  | '7xl'
+  | '8xl'
+  | 'full'
+  | `container.${'sm' | 'md' | 'lg' | 'xl' | '2xl'}`
+  | 'auth0-width';
```

Comment:
This means we need to keep the two definitions in sync. 
You can leverage `keyof typeof` instead:

```suggestion
type ContainerSize = keyof typeof sizes.container;
export type Size = Exclude<keyof typeof sizes, 'container'> | `container.${ContainerSize}`;
```